### DISCUSSION TITLE
Can't push docker image to ghrc.io

### DISCUSSION THREAD

**Discussion by MichaelBulmerHollowayControls** [community/community#67718] [Packages]
### Select Topic Area

Question

### Body

Steps taken:
1. docker login ghrc.io  ->  Succesful logging in with PAT
2. docker push ghrc.io/michaelbulmerhollowaycontrols/helixpostgres:0.1 
-> 
Leads to repeated retries ie:
`
bfb61fa044c5: Retrying in 1 second`
and eventual 
`received unexpected HTTP status: 200 OK`

Due to similar issues mentioning proxies have tried on different network (mobile hotspot and fibre connection) and on different machines.
Have tried different combinations of ghrc.io/{_**namespace**_}/helixpostgres:0.1 
i.e. My username and organisation.

Top muppetry on my part or a bug?

**Comment by MichaelBulmerHollowayControls** [comment]
muppetry

**Reply by scottschreckengaust** [reply]
What was the problem? Running into "same" issue. It has:

>  Retrying in 3 seconds

then responds with a:

> received unexpected HTTP status: 200 OK

**Comment by MichaelBulmerHollowayControls** [comment]
@scottschreckengaust   I had a typo in the url.  ghrc.io no ghcr.io

**Reply by Nano204** [reply]
Hahaha I hate typos. When you re down, the bug looks like not fixable and you're thinking of quitting as a developer... Yes... It was just a typo.

Thanks! I had the same typo.

**Comment by scottschreckengaust** [comment]
Use correct URL
